plugins/hydra_submitit_launcher/hydra_plugins/hydra_submitit_launcher/config.py

```diff
@@ -45,27 +63,12 @@ class SlurmQueueConf:
 
 
 @dataclass
-class LocalQueueConf:
-    # local executor mocks the behavior of slurm locally
-
-    # maximum time for the job in minutes
-    timeout_min: int = 60
-    # number of gpus to use on each node
-    gpus_per_node: int = 1
-    # number of tasks to spawn on each node (only one node available in local executor)
-    tasks_per_node: int = 1
-
-
-@dataclass
-class AutoQueueConf:
-    # auto executor automatically identifies and uses available cluster
-    # Currently this is only slurm, but local executor can be manually forced
-    # instead.
-    # Most parameters are shared between clusters, some can be cluster specific
+class SubmititConf(BaseSubmititConf):
```

Comment:
This is the main configuration, it provides all the default values, and can be overriden by executor specific configs